src/main/java/Factory/ContactTypeFactory.java

```diff
@@ -0,0 +1,23 @@
+package Factory;
```

Comment:
/**
 * @author Daniella Burgess 219446482
 * This is ContactTypeFactory.java
 * Date 07 April 2022
 */